pypeit/scripts/ql_keck_mosfire.py

```diff
@@ -0,0 +1,377 @@
+#!/usr/bin/env python
+#
+# See top-level LICENSE file for Copyright information
+#
+# -*- coding: utf-8 -*-
+"""
+This script runs PypeIt on a pair of NIRES images (A-B)
+"""
+import argparse
+
+from pypeit import msgs
+import os
+import sys
+import numpy as np
+import copy
+from astropy.io import fits
+from IPython import embed
+
+from pypeit import pypeit
+from pypeit import par, msgs
+from pypeit import pypeitsetup
+from pypeit import wavecalib
+from pypeit import wavetilts
+from pypeit import spec2dobj
+from pypeit import coadd2d
+from pypeit.core import framematch
+from pypeit import specobjs
+from pypeit.images import buildimage
+from pypeit import slittrace
+from pypeit.spectrographs.util import load_spectrograph
+from pypeit import reduce
+from pypeit import calibrations
+from pypeit import ginga
+from pypeit.core.parse import get_dnum
+from astropy.stats import sigma_clipped_stats
+import warnings
```

Comment:
Please organize the imports: https://www.python.org/dev/peps/pep-0008/#imports

Reply:
Done.